examples/rules.mk

```diff
@@ -92,7 +92,8 @@ endif
 include $(OPENCM3_DIR)/mk/genlink-config.mk
 endif
 
-SCRIPT_DIR	= $(OPENCM3_DIR)/scripts
+OPENCM3_SCRIPT_DIR = $(OPENCM3_DIR)/scripts
+EXAMPLES_SCRIPT_DIR	= $(OPENCM3_DIR)../scripts
```

Comment:
in one of them, you use an intermediate, /, but not the otther, is this really intentional?